Symfony/CS/Fixer/PSR2/BracesFixer.php

```diff
@@ -605,4 +606,27 @@ private function getFinalControlContinuationTokensForOpeningToken($openingTokenK
 
         return array();
     }
+
+    private function fixDeclareStrictTypes(Tokens $tokens)
+    {
+        $declares = $tokens->findGivenKind(T_DECLARE);
+
+        foreach ($declares as $index => $declare) {
+            if ($this->isStrictTypeDeclare($index, $tokens)) {
```

Comment:
I would go to direction of PSR12 proposal.
https://github.com/php-fig/fig-standards/blob/master/proposed/extended-coding-style-guide.md#strict-types-namespace-and-use-declarations

So form without braces as you show.

The problem is that there is no standard for form with braces (or at least I can't find them), ref https://github.com/cs-extended/fig-standards/issues/24